### PR TITLE
Fix typos and suggest phrasing.

### PR DESCRIPTION
**Pull request by leonardovega**
Just fixed a few typos and grammatical errors. Added a little to the teaching material.

### CODE REVIEW

File: new_course/1_programming_fundementals/debugging.md [hunk 1]
```diff
@@ -24,13 +24,13 @@ It looks something like this:
...
 
 <a href="https://i.imgur.com/5LJDp9v.png"><img class="tutorial-img" src="https://i.imgur.com/5LJDp9v.png" title="source: imgur.com" /></a>
 
 First, this line of the stack trace will tell you what specific line caused the runtime error. In the above example, the error was encountered in line 30 of `lib/bottles.rb`. This line will also provide a brief explanation of the error, as well as the name of error, (in this case, [`NameError`](https://ruby-doc.org/core-2.6/NameError.html)). And yes, Ruby [errors](https://ruby-doc.org/core-2.6/Exception.html) are *also* objects.
 
-And there you have it. The stack trace really is that simple. At this point, you know where in your code the exception is being raised, and you know the kind of error you're dealing with. You might even know what fixes need to be implemented.
+Well, there you have it. The stack trace really is that simple. At this point, you know where in your code the exception is being raised, and you know the kind of error you're dealing with. You might even know what fixes need to be implemented.
```
Comment (leonardovega) [inline new_course/1_programming_fundementals/debugging.md]: This was just a preference, since the line before started with "And" as well.

**Comment by leila-alderman** [comment]
Oops! Looks like we were both working on the same file! I just merged in https://github.com/TheOdinProject/curriculum/pull/9863, so now there are merge conflicts here.

I don't want to make you do duplicate work, but feel free to take a pass over the more recent version. Let me know if you'd like to resolve the merge conflicts on this PR or just close it (or open a new PR).

In general, it's a good idea to glance over the open PRs before working on one of the lessons. (We've run into this problem before, so it's not just you!)

File: new_course/1_programming_fundementals/debugging.md [hunk 2]
```diff
@@ -128,13 +128,15 @@ def double_words_in_phrase(string)
...
+Using Pry to debug achieves the same outcome as `puts` debugging: it allows you to confirm the assumptions you have about particular parts of your code. If your code is complex, Pry will probably allow you to debug quicker thanks to its interactive, runtime environment. Using Pry allows you to check your code while getting instant feedback. Whereas using `puts` debugging requires you to add many `puts` statements and re-run your code everytime you want to check for something new. Plus, with `puts` debugging you have to remember to go back and remove all those `puts` statements from you code.   
 
-There is a lot, lot more you can do with Pry which is out of the scope of this lesson. Check out the assignments and extra resources to find out where you can learn more about this useful gem.
+There is a lot, a lot more you can do with Pry which is out of the scope of this lesson. Check out the assignments and extra resources to find out where you can learn more about this useful gem.
```
Comment (leonardovega) [inline new_course/1_programming_fundementals/debugging.md]: I thought you were trying to say... "There is a lot <pause> a lot more you can do with..." but looking back I think the original statement was saying something similar to: "There are lots lots more you can do with...".

**Comment by leonardovega** [comment]
What a coincidence! :smile: I'll go through the lesson again and delete my branch. Thanks for the advice. In the future though, what's the best way to avoid having two people working on the same file before either one submits a pull request?

**Comment by leila-alderman** [comment]
In this particular case, I opened up the PR on Saturday, so it's likely that it was there before you started editing. In general, though, since there aren't that many of us working on the lessons, we can coordinate "checking out" lessons in the #ft-ruby-basics-lessons channel on Discord. (If you don't have write permission there, let me know.)

Thanks for your work on these lessons!

**Pull request closed without merging**